pkgs/tools/networking/tcpdump/default.nix

```diff
@@ -2,21 +2,13 @@
 
 stdenv.mkDerivation rec {
   pname = "tcpdump";
-  version = "4.9.3";
+  version = "4.9.99";
```

Comment:
Well, that doesn't sound like a stable release.  Any evidence that it's suitable for nixpkgs master?

Reply:
https://www.tcpdump.org/index.html#latest-releases

Which means we need to keep the patch around for a while longer.

Reply:
Hum yes I wanted to pull the 4.99.0 release not the non existing 4.9.99 :/

Reply:
Updated the PR to use 4.99.0, sorry for the confusion.

Reply:
Well, at a glance 4.99.0 doesn't sound stable either :-)

Reply:
4.99.0 is as far as I can tell. It has a [proper tag in their repo](https://github.com/the-tcpdump-group/tcpdump/releases/tag/tcpdump-4.99.0). [Debian Testing](https://packages.debian.org/bullseye/tcpdump), [Fedora 32 and Fedora 33](https://src.fedoraproject.org/rpms/tcpdump) are all using the v4.99.0 so it should be stable enough for nixpkgs. :)